appveyor.yml

```diff
@@ -0,0 +1,53 @@
+version: "{branch}-ci-{build}"
+os:
+- Visual Studio 2013
+
+platform:
+- Win32
+
+configuration:
+- Debug 
+#- Release
+
+on_finish:
+#- ps: $blockRdp = $true; iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
+
+environment:
+  matrix:
+    - BOOST_ROOT: 'C:\Libraries\boost'
+      BOOST_INCLUDEDIR: 'C:\Libraries\boost\boost'
+      BOOST_LIBRARYDIR: 'C:\Libraries\boost\lib32-msvc-12.0'
+      ruby_version: "193"
+    - BOOST_ROOT: 'C:\Libraries\boost_1_58_0'
+      BOOST_INCLUDEDIR: 'C:\Libraries\boost_1_58_0\boost'
+      BOOST_LIBRARYDIR: 'C:\Libraries\boost_1_58_0\lib32-msvc-12.0'
+      ruby_version: "200"
```

Comment:
I've found it: 



>  I'm afraid Cucumber on Windows/Ruby requires ruby 2.0.0 (x86). 
>  The reason is that the gherkin gem doesn't yet ship with compiled binaries for more recent versions or Ruby, and not for x64.
> 
>  We're working on a Gherkin3 which will address this issue. 
>  See this post for background. I cannot give you an ETA, but it's at least a few months away.

[source](http://stackoverflow.com/questions/29507570/cannot-load-such-file-2-2-gherkin-lexer-en-how-to-fix)